doc/man3/DH_get0_pqg.pod

```diff
@@ -57,10 +57,7 @@ after this function has been called. The I<q> parameter may be NULL.
 DH_set0_pqg() also checks if the parameters associated with I<p> and I<g> and
 optionally I<q> are associated with known safe prime groups. If it is a safe
 prime group then the value of I<q> will be set to q = (p - 1) / 2 if I<q> is NULL.
-For safe prime groups the optional length parameter I<length> is set to twice
-the value of the maximum_target_security_strength(BN_num_bits(I<p>)) as listed in
-SP800-56Ar3 Table(s) 25 & 26. If it is not a safe prime group then the optional
-length parameter will be set if I<q> is not NULL to BN_num_bits(I<q>).
+The optional length parameter will be set if I<q> is not NULL to BN_num_bits(I<q>).
```

Comment:
Ugh.   What about:

`The optional length parameter will be set to BN_num_bits(I<q>) if I<q> is not NULL`

or

`If I<q> is not NULL, the optional length parameter will be set to BN_num_bits(I<q>)`

Reply:
Ok..